packages/react-components/react-tree/stories/Tree/Virtualization.md

```diff
@@ -0,0 +1,3 @@
+A `Tree` can be virtualized to optimize the rendering of a large tree structure. This technique involves creating a virtual version of the tree that only loads the visible nodes at any given time. By doing so, it reduces the number of DOM nodes that need to be rendered, which in turn improves the overall performance of the tree.
```

Comment:
```suggestion
A `Tree` can be virtualized to optimize the rendering of a large tree structure. This technique involves creating a virtual version of the tree that only loads the visible nodes at any given time. By doing so, it reduces the number of DOM nodes that need to be rendered, which in turn improves time to interaction of large trees.
```

 Virtualization is actually a performance bottleneck, it increases memory and CPU usage and most of the metrics involving it are actually worse than simply not using virtualization. We should emphasise that the usage of virtualization should be sparring and it should only be implemented when TTI problems are spotted, since that is the main advantage metric in favor of virtualization  https://hackmd.io/@bsunderhus/rkDnTIhQ5

Reply:
oh got it, thanks for the explanation! 👍